connection-builder.d.ts

```diff
@@ -11,4 +11,14 @@ export class ConnectionBuilder {
      * Starts a new process and takes over its stdin and stdout. Returns a Connection object that can be used to communicate with the new process 
      */
     build(): Connection;
+
+    /**
+     * Available to override and manage onExit by your own implementation
+     */ 
+    onExited: (code: any) => void;
+   
+    /**
+     * Available to override and manage errors by your own implementation 
```

Comment:
Can we make it explicit in the comment that this will be what's on the stderr without any treatment?
The reason why I'm bringing this up is that what is send from the .net client might arrive here in "chunks". Say there's a `Console.Error.WriteLine("Really long content...")` in .net, the onError callback might be called with "Really lon" and then again with "g content...".

I was also going to suggest changing the name from onError to onStderr, but I don't think that makes it any more clear.